### ISSUE TITLE
GNN PSR benchmark

### ISSUE THREAD

**Issue by swenson-nick** [drorlab/atom3d#41]
When I trained the GNN model provided in examples/psr/gnn/model.py using the hyperparameters specified in examples/psr/gnn, I'm getting a per-target Spearman's rho of 0.503 +/- 0.013 on the validation set, about what is reported in the paper. However, I'm only getting a per-target Spearman's rho of 0.405 +/- 0.013 on the test set. For reference, I'm seeing 0.582 +/- 0.007 for the per-target Spearman on the training data after 50 epochs.

**Comment by awfderry** [comment]
Thanks for bringing this to our attention! We identified the issue and updated accordingly (#45).
